h2o-docs/src/product/mojo-quickstart.rst

```diff
@@ -693,6 +693,54 @@ If you cannot install Graphviz on your environment, another option is to produce
   :alt: Example Output Picture
   :scale: 50%
 
+Printing GBM MOJOs
+~~~~~~~~~~~~~~~~~~
+
+The ``print_mojo`` function allows a GBM model to be coverted to `ONNX <https://onnx.ai/about.html>`__ representation, which means the model will be rendered in an "open format built to represent" ML models. This allows you to use the model with a variety of frameworks, tools, runtimes, and compilers. 
```

Comment:
this is still innacurate, the output has nothing to do with ONNX, also this works for all tree-like models

I would reduce this to

"The ``print_mojo`` function allows a model to be converted to machine readable representation - either to JSON for further processing, dot for rendering images.

@koniecsveta maybe we should update print_mojofunction to also allow PNG output?

Reply:
yesyes, I think it will be an easy thing after adding it to flow.